cmd/earth/main.go

```diff
@@ -1532,6 +1596,11 @@ func defaultConfigPath() string {
 }
 
 func getInstallID() (string, error) {
+	installID, ok := os.LookupEnv("EARTHLY_INSTALL_ID")
+	if ok {
+		return installID, nil
+	}
```

Comment:
I'm thinking we could also try to craft an install ID from repo + some build name, when CI is true. Build name would be workflow name / pipeline name / build name - something that stays constant for each CI build pipeline.

And when CI is false, we can use the generated file in `.earthly` dir.

Reply:
excellent plan, I just pushed up a new version to do this.